### ISSUE TITLE
How to display the line number？

### ISSUE THREAD

**Issue by Xuzan9396** [kpango/glg#103]
How to display the line number？

**Comment by Xuzan9396** [comment]
For example, the official package
```
log.SetFlags(log.Lshortfile | log.LstdFlags)
```

**Comment by Xuzan9396** [comment]
![image](https://user-images.githubusercontent.com/24741439/123052611-99958400-d435-11eb-8e19-604e15be612c.png)

**Comment by Xuzan9396** [comment]
Solve Your Way

```
	glg.Get().
		SetMode(glg.BOTH). // default is STD
		SetPrefix(glg.PRINT,getCaller(0)).
		SetPrefix(glg.LOG,getCaller(0)).
		SetPrefix(glg.INFO,getCaller(0)).
		SetPrefix(glg.WARN,getCaller(0)).
		SetPrefix(glg.ERR,getCaller(0)).
		SetPrefix(glg.WARN,getCaller(0)).
...

func getCaller(skip int) (string) {
	_, file, line, ok := runtime.Caller(skip)
	//fmt.Println(file)
	//fmt.Println(line)
	if !ok {
		return ""
	}
	n := 0
	for i := len(file) - 1; i > 0; i-- {
		if file[i] == '/' {
			n++
			if n >= 2 {
				file = file[i+1:]
				break
			}
		}
	}
	return file +":"+ strconv.Itoa(line)
}
```

**Comment by Milerius** [comment]
same question here @kpango

**Comment by kpango** [comment]
Sorry for replying so late.
I just released glg v1.6.1 and created new configuration interfaces called `SetTraceLineFlag` and `SetLevelTraceLineFlag`. 
Use `SetTraceLineFlag` if you want to set the global flag of glg, and use `SetLevelTraceLineFlag` if you want to set the trace settings for a specific level.

example code is below
https://github.com/kpango/glg/blob/master/example/main.go#L173-L182
